src/witan/app/user.clj

```diff
@@ -11,25 +12,25 @@
   (let [randomdata (nonce/random-bytes 16)]
     (codecs/bytes->hex randomdata)))
 
-(defn find-user [username]
+(defn find-user-by-username [username]
   (hayt/select :Users (hayt/where {:username username})))
 
-(defn create-user [user password]
-  (let [hash (hs/encrypt password)]
-    (hayt/insert :Users, (hayt/values :username user :password_hash hash))))
+(defn create-user [user]
```

Comment:
As `user` isn't passed on, I think it's worth key destructuring here `[{:keys [username password name]}]`